Dockerfile

```diff
@@ -2,82 +2,63 @@
 # Zulip development environment image and use
 # tools/build-release-tarball to generate a production release tarball
 # from the provided Git ref.
-FROM ubuntu:xenial-20171114
-LABEL maintainer="Alexander Trost <galexrt@googlemail.com>"
+FROM ubuntu:18.04 as base
 
-# You can specify these in docker-compose.yml or with
-#   docker build --build-args "ZULIP_GIT_REF=git_branch_name" .
-ARG ZULIP_GIT_URL=https://github.com/zulip/zulip.git
-ARG ZULIP_GIT_REF=2.0.4
-ARG CUSTOM_CA_CERTIFICATES=
+# Set up working locales and upgrade the base image
+ENV LANG="C.UTF-8"
 
-SHELL ["/bin/bash", "-xuo", "pipefail", "-c"]
```

Comment:
Why is it desirable to remove these shell options?  I seem to recall them being useful for debugging.

Reply:
We have no pipes anymore, so it’s mostly just reducing noise and conforming to convention (`SHELL` is passed to inherited Dockerfiles).